__tests__/app.test.js

```diff
@@ -42,7 +42,7 @@ describe("ERROR Handling", () => {
         expect(body.msg).toBe("Endpoint was not found!");
       });
   });
-  test("status: 400 for an invalid article_id", () => {
+  test("status: 404 for an invalid article_id", () => {
```

Comment:
this should be a 400 as it is an invalid ID rather than just an ID that does not exist